prisma/schema.prisma

```diff
@@ -0,0 +1,65 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model User {
+  id            String         @id @default(uuid())
+  email         String         @unique
+  profile       Profile? //user have profile //relations  
+  githubDetails GitHubDetails? //user have gitHub //relations
+}
+
+model Profile {
+  id             String         @id @default(uuid())
+  user           User           @relation(fields: [userId], references: [id])
+  userId         String         @unique
+  fullName       String
+  email          String
+  linkedIn       String
+  bio            String
+  country        Country        @relation(fields: [countryId], references: [id])
+  countryId      String
+  city           City           @relation(fields: [cityId], references: [id])
+  cityId         String
+  remoteOnly     Boolean
+  position       String
+  seniority      String
+  techStack      String[]
+  employmentType EmploymentType
+  isPublished    Boolean
```

Comment:
Can you please add default value to this column, so it's: `isPublished    Boolean  @default(false)`